plugins/postcss-initial/src/properties.ts

```diff
@@ -0,0 +1,2 @@
ight-radius','0'],['border-bottom-style','none'],['border-bottom-width','medium'],['border-collapse','separate'],['border-image','none'],['border-left','0'],['border-left-color','currentcolor'],['border-left-style','none'],['border-left-width','medium'],['border-radius','0'],['border-right','0'],['border-right-color','currentcolor'],['border-right-style','none'],['border-right-width','medium'],['border-spacing','0'],['border-top','0'],['border-top-color','currentcolor'],['border-top-left-radius','0'],['border-top-right-radius','0'],['border-top-style','none'],['border-top-width','medium'],['bottom','auto'],['box-shadow','none'],['box-sizing','content-box'],['caption-side','top'],['clear','none'],['clip','auto'],['color','#000'],['columns','auto'],['column-count','auto'],['column-fill','balance'],['column-gap','normal'],['column-rule','medium none currentcolor'],['column-rule-color','currentcolor'],['column-rule-style','none'],['column-rule-width','medium'],['column-span','1'],['column-width','auto'],['content','normal'],['counter-increment','none'],['counter-reset','none'],['cursor','auto'],['display','inline'],['empty-cells','show'],['float','none'],['font-family','serif'],['font-size','medium'],['font-style','normal'],['font-variant','normal'],['font-weight','normal'],['font-stretch','normal'],['line-height','normal'],['height','auto'],['hyphens','none'],['left','auto'],['letter-spacing','normal'],['list-style','disc outside none'],['list-style-image','none'],['list-style-position','outside'],['list-style-type','disc'],['margin','0'],['margin-bottom','0'],['margin-left','0'],['margin-right','0'],['margin-top','0'],['max-height','none'],['max-width','none'],['min-height','0'],['min-width','0'],['opacity','1'],['orphans','2'],['outline','medium none currentcolor'],['outline-color','currentcolor'],['outline-style','none'],['outline-width','medium'],['overflow','visible'],['overflow-x','visible'],['overflow-y','visible'],['padding','0'],['padding-bottom','0'],['padding-left','0'],['padding-right','0'],['padding-top','0'],['page-break-after','auto'],['page-break-before','auto'],['page-break-inside','auto'],['perspective','none'],['perspective-origin','50% 50%'],['position','static'],['quotes','“ ” ‘ ’'],['right','auto'],['tab-size','8'],['table-layout','auto'],['text-align','left'],['text-align-last','auto'],['text-decoration','none'],['text-decoration-color','inherited'],['text-decoration-color','currentcolor'],['text-decoration-line','none'],['text-decoration-style','solid'],['text-indent','0'],['text-shadow','none'],['text-transform','none'],['top','auto'],['transform','none'],['transform-origin','50% 50% 0'],['transform-style','flat'],['transition','none 0s ease 0s'],['transition-delay','0s'],['transition-duration','0s'],['transition-property','none'],['transition-timing-function','ease'],['vertical-align','baseline'],['visibility','visible'],['white-space','normal'],['widows','2'],['width','auto'],['word-spacing','normal'],['z-index','auto']]);
```

Comment:
The old plugin did a lot of templating/formatting at runtime.

This is now all pre-computed, so the plugin is smaller, faster and much simpler.

Because I am not using weird file/folder names this bug is also gone : https://github.com/maximkoretskiy/postcss-initial/issues/51